<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageTools</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/tool_icon/resource/main.png</normaloff>:/tool_icon/resource/main.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QMdiArea" name="mdiArea">
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="viewMode">
       <enum>QMdiArea::TabbedView</enum>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="tabsMovable">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1140</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="field_depth_tool"/>
    <addaction name="shake_tool"/>
    <addaction name="imageeditor"/>
    <addaction name="af_calc_tool"/>
    <addaction name="rawimageeditor"/>
   </widget>
   <widget class="QMenu" name="help_button">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="userguide"/>
   </widget>
   <addaction name="menu_2"/>
   <addaction name="help_button"/>
  </widget>
  <action name="field_depth_tool">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>镜头计算器</string>
   </property>
  </action>
  <action name="shake_tool">
   <property name="text">
    <string>抖动测试工具</string>
   </property>
  </action>
  <action name="imageeditor">
   <property name="text">
    <string>图片分析工具</string>
   </property>
  </action>
  <action name="af_calc_tool">
   <property name="text">
    <string>镜头曲线计算工具</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="userguide">
   <property name="text">
    <string>用户手册</string>
   </property>
  </action>
  <action name="rawimageeditor">
   <property name="text">
    <string>raw图分析工具</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
